src/pages/console/tutorial/extend-cli.mdx

```diff
@@ -22,7 +22,34 @@ import all1 from "/src/fragments/pull-cli-studio.mdx";
 
 <Fragments fragments={{all: all1}} />
 
-## Adding features with the Amplify CLI
+<Callout>
+
+For Safari browser, the browser will provide a CLI login key. Copy the CLI login key from the browser and paste it into your terminal.
+
+</Callout>
+
+## Use component in code
+
+Now that your components are synced, navigate to a component and choose **Configure**. To get your React code, choose **Get component code**.
```

Comment:
```suggestion
Now that your components are synced, navigate to a component and choose **Configure**. To get your React code, choose **Get component code**.
```

Are their multiple options on a component page that would require us to choose? Or can we swap this for "select"? https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#15-use-carets-to-help-users-navigate-menus-use-select-instead-of-click

Reply:
done